### DISCUSSION TITLE
Got Error: Cannot read properties of null (reading 'trim') when build with 34.1.8

### DISCUSSION THREAD

**Discussion by thachphongphong** [vercel/vercel#11546] [General] [answered]
### Description

My code is built ok with 34.1.7, but when i update to 34.1.8 i got this error.
Got Error: Cannot read properties of null (reading 'trim')

**Comment by FredrikAugust** [comment, marked as answer]
An issue has been created here: https://github.com/vercel/vercel/issues/11547.
